### ISSUE TITLE
[Bug Report][2.3.23] aria-expanded in v-expansion-panel

### ISSUE THREAD

**Issue by tingrt** [vuetifyjs/vuetify#16099]
### Environment
**Vuetify Version:** 2.3.23
**Vue Version:** 2.6.11
**Browsers:** Chrome 106.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
go to https://vuetifyjs.com/en/components/expansion-panels/#accordion
open the dev tools
open in the usage sample the first item
select this item with the element inspector to see the generated html structure.

### Expected Behavior
According to https://www.w3.org/TR/wai-aria-practices-1.1/#accordion the "header button element has aria-expanded set to true".

### Actual Behavior
The parent div element has a property aria-expanded set to true. This doesn't work for all screenreaders.

### Reproduction Link
[https://codepen.io/clausstraube/pen/BaoamWP?editors=1010](https://codepen.io/clausstraube/pen/BaoamWP?editors=1010)

**Comment by tingrt** [comment]
Can we fix this issue in the vuetify2 as we are using the vue2

**Comment by KaelWD** [comment]
Duplicate of #15514
